lk2nd/device/dts/msm8994/msm8992-lg-h815.dts

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8992 0>, <252 0>;
+	qcom,board-id = <0xb64 0>;
+	qcom,pmic-id = <0x10009 0x1000a 0x00 0x00>;
+};
+
+&lk2nd {
+	model = "LG G4 (International)";
+	compatible = "lg,h815";
```

Comment:
If there is some (close-to-)mainline dts for this device, could you add an `lk2nd,dtb-hints = ` property like here:

https://github.com/msm8916-mainline/lk2nd/blob/f47144913fe17851e0e628729ddf6a061cff46e0/lk2nd/device/dts/msm8916/apq8016-sbc.dts#L15

or, if there is none at the moment, "guess" it and add a fixme like:

https://github.com/msm8916-mainline/lk2nd/blob/f47144913fe17851e0e628729ddf6a061cff46e0/lk2nd/device/dts/msm8916/msm8916-asus-z00e.dts#L15

Reply:
I have a dt for this device that isn't upstreamed currently but I will someday. Should I make it a comment or let it as is?

Reply:
@FuchtelJockel +1 for this PR.
BTW what blocks you to send DTS patch to mainline kernel? Because now it will be on second downstream project. Feel free to Cc me when you send the patch.

Reply:
> I have a dt for this device that isn't upstreamed currently but I will someday. Should I make it a comment or let it as is?

If you have some own close-to-mainline tree and you don't plan on changing the name of the file while upstreaming it, add as is.